packages/sveltekit/src/utils/supabase-load.ts

```diff
@@ -1,18 +1,18 @@
 import { getConfig } from '../config';
-import { isBrowser } from '@supabase/auth-helpers-shared';
 import { createClient, type Session } from '@supabase/supabase-js';
 import type { LoadEvent } from '@sveltejs/kit';
 import type { TypedSupabaseClient } from '../types';
 
 export function getLoadSupabaseClient(event: LoadEvent): TypedSupabaseClient {
-  const { supabaseUrl, supabaseKey, options, globalInstance } = getConfig();
-
-  if (isBrowser()) {
-    return globalInstance;
-  }
```

Comment:
I'm aware this will cause each load function to new up a new client instance, but I think the trade-off is worth it here, considering we are removing requests the client has to do.